src/main/app/claims/models/reviewOrder.ts

```diff
@@ -21,4 +21,12 @@ export class ReviewOrder {
 
     return this
   }
+
+  isReconsiderationDeadlinePassed (deadline: moment.Moment): boolean {
+    if (!deadline) {
+      return false
```

Comment:
just a thought, if i am using this method and if it returns false, i may think its returning false because reconsideration deadline has not passed but in fact it will return false even if deadline is undefined.